cmd/nerdctl/container/container_commit_test.go

```diff
@@ -36,14 +36,7 @@ func TestCommit(t *testing.T) {
 				helpers.Anyhow("rmi", "-f", data.Identifier())
 			},
 			Setup: func(data test.Data, helpers test.Helpers) {
-				// FIXME: short of pulling first, docker will fail to start the container.
-				// Debugging shows container exited immediately. Nothing in the container logs. Not much in journalctl.
-				// It is not clear what is happening.
-				if nerdtest.IsDocker() {
-					helpers.Ensure("pull", testutil.CommonImage)
-				}
```

Comment:
This change doesn't seem relevant to top?

Reply:
Correct.
top_test was suffering from the same issue, which I fixed in a separate PR, and I ended-up removing both workarounds at the same time.
Lmk of you want another commit for this.

Reply:
Yes, another commit is more preferable

Reply:
Done.